packages/embed/src/factories/create-popover/create-popover.ts

```diff
@@ -97,6 +97,47 @@ const buildTooltip = (message: string, hide: () => void) => {
   return container
 }
 
+const LOCAL_STORAGE_NOTIFICATION_DATA = 'tfNotificationData'
+
+type LocalStorageCUIData = {
+  formId: string
+  expireDate: number
+}
+const today = new Date().getTime()
+
+const cuiLocaslStorageData = localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
+
+const getLocalStorageData = () => cuiLocaslStorageData && JSON.parse(cuiLocaslStorageData)
```

Comment:
Typo in var name. I would move it into `getLocalStorageData` method if possible:

```suggestion
const getLocalStorageData = () => {
  const data = localStorage.getItem(LOCAL_STORAGE_NOTIFICATION_DATA)
  return data && JSON.parse(data)
}
```

Reply:
I think this will also solve the issue with server-side rendering (failing build for nextjs demo)

Reply:
getItem returns <T> | null so you can feed that to JSON.parse and not have to check here, otherwise this function returns false | Object